app/views/spree/shared/_paypal_checkout_button.html.erb

```diff
@@ -1,18 +1,7 @@
-<% address = current_order.ship_address %>
-
 <div id="paypal-button"></div>
 
 <script>
-  var address = {
-    line1: '<%= address.address1 %>',
-    line2: '<%= address.address2 %>',
-    city: '<%= address.city %>',
-    state: '<%= address.state.name %>',
-    postalCode: '<%= address.zipcode %>',
-    countryCode: '<%= address.country.iso %>',
-    phone: '<%= address.phone %>',
-    recipientName: '<%= "#{address.firstname} #{address.lastname}" %>'
-  }
+  var address = <%== SolidusPaypalBraintree::Address.new(current_order.ship_address).to_json %>
```

Comment:
There should be no reason for `raw` output for JSON.

```suggestion
  var address = <%= SolidusPaypalBraintree::Address.new(current_order.ship_address).to_json %>
```

should work as well.